src/internal/HtmlColorParse.cpp

```diff
@@ -0,0 +1,54 @@
+/*-------------------------------------------------------------------------
```

Comment:
Arduino doesn't have LTO (Link Time Optimization) enabled yet. Therefore, it just links the full source code or skips it completely. It can't just link some functions and remove others. At least not yet. Therefore, we need to put different methods in different files to allow the linker to optimize out unused methods.